test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -45,6 +46,14 @@ TEST(QuicPlatformTest, QuicArraysize) {
 
 enum class TestEnum { ZERO = 0, ONE, TWO, COUNT };
 
+TEST(QuicPlatformTest, QuicBugTracker) {
+  // Just make sure they compile.
+  QUIC_BUG << "Here is a bug";
```

Comment:
For my own knowledge, why does QUIC_BUG not abort here?

Reply:
Nothing surprising here, it does abort in debug mode. My intention is just to check these statements can compile, please take a look at the latest commit.

Sorry for the stupidness. I must have context switched between writing this test and creating the PR.